src/OpenDebugAD7/AD7Resources.resx

```diff
@@ -192,6 +192,9 @@
   <data name="Error_UnableToSetBreakpoint" xml:space="preserve">
     <value>Error setting breakpoint. {0}</value>
   </data>
+  <data name="Error_UnableToSetTracepoint" xml:space="preserve">
```

Comment:
use a name that describes the error message you are giving, not the action `Error_UnableToParseLogMessage`